remote-query/remote-query-client/src/main/java/org/infinispan/query/remote/client/impl/QueryRequest.java

```diff
@@ -98,7 +109,8 @@ public Json toJson() {
             .set(QUERY_STRING_FIELD, queryString)
             .set(START_OFFSET_FIELD, startOffset)
             .set(MAX_RESULTS_FIELD, maxResults)
-            .set(NAMED_PARAMETERS_FIELD, Json.make(getNamedParameters()));
+            .set(NAMED_PARAMETERS_FIELD, Json.make(getNamedParameters()))
+            .set(LOCAL_FIELD, Json.make(local));
```

Comment:
can't you use `bool()` directly?
```suggestion
            .set(LOCAL_FIELD, Json.factory().bool(local));
```
`Json.make()` seems quite a large method full of `if-else-if` xD